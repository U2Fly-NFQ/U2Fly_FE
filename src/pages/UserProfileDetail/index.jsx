import React from 'react'
import { Col, Row } from 'antd'

function UserProfileDetail(props) {
  return (
    <Row className="userProfile-container-profile">
      <Col span={24} className="userProfile-container-profile-title">
        My Profile
      </Col>
      <Col span={24}></Col>
    </Row>
  )
}

export default UserProfileDetail
