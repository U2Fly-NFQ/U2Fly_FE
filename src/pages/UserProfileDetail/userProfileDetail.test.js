import React, { Suspense } from 'react'
import { BrowserRouter as Router } from 'react-router-dom'

import UserProfileDetail from './'
import renderer from 'react-test-renderer'

describe('User profile detail', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Router>
          <Suspense>
            <UserProfileDetail />
          </Suspense>
        </Router>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
