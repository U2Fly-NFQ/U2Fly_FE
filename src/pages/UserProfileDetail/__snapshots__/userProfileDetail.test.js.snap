// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User profile detail renders correctly 1`] = `
<div
  className="ant-row userProfile-container-profile"
  style={Object {}}
>
  <div
    className="ant-col ant-col-24 userProfile-container-profile-title"
    style={Object {}}
  >
    My Profile
  </div>
  <div
    className="ant-col ant-col-24"
    style={Object {}}
  />
</div>
`;

exports[`User profile detail renders correctly 1`] = `
<div
  className="ant-row userProfile-container-profile"
  style={Object {}}
>
  <div
    className="ant-col ant-col-24 userProfile-container-profile-title"
    style={Object {}}
  >
    My Profile
  </div>
  <div
    className="ant-col ant-col-24"
    style={Object {}}
  />
</div>
`;
